### DISCUSSION TITLE
Nest can't resolve dependencies of the SendNotificationCommand's InquirerService

### DISCUSSION THREAD

**Discussion by simplenotezy** [jmcdo29/nest-commander#824] [Q&A] [answered]
How do I enable the `InquirerService`? I have been working with it for quite a while, but all of a sudden, I am now getting below error message.

I don't seem to be able to find any documentation on how to enable `InquirerService`, e.g. if it should be added as a provider etc to `CliModule`

```
[Nest] 84901  - 03/10/2023, 6:16:04 PM   ERROR [LOCAL ExceptionHandler] Nest can't resolve dependencies of the SendNotificationCommand (NotificationService, DatingProfileService, ?, DataSource). Please make sure that the argument InquirerService at index [2] is available in the CliModule context.

Potential solutions:
- Is CliModule a valid NestJS module?
- If InquirerService is a provider, is it part of the current CliModule?
- If InquirerService is exported from a separate @Module, is that module imported within CliModule?
  @Module({
    imports: [ /* the Module containing InquirerService */ ]
  })

**Comment by jmcdo29** [comment, marked as answer]
So long as this goes through `CommandFactory`, that shouldn't be an error you're getting. That provider comes from the `CommandRunnerModule.forRoot()` call, which is made by `CommandFactory.run()`

**Reply by simplenotezy** [reply]
Thanks @jmcdo29 that solved it!
